core/memslot.c

```diff
@@ -362,14 +362,12 @@ static inline void memslot_delete(hax_memslot *dest)
     hax_vfree(dest, sizeof(hax_memslot));
 }
 
-static inline void memslot_move(hax_memslot *dest, hax_memslot *src)
-{
-    hax_list_node entry;
-
-    ramblock_deref(dest->block);
-    entry = dest->entry;
-    memslot_init(dest, src);
-    dest->entry = entry;
+static inline void memslot_move(hax_memslot *dest, hax_memslot *src)
+{
+	ramblock_deref(dest->block);

```

Comment:
According to the current coding style, it is recommended to use 4 spaces as indent.